lib/cc/engine/analyzers/javascript/minification_checker.rb

```diff
@@ -10,8 +10,10 @@ def initialize(path)
           end
 
           def minified?
-            ratio = content.chars.count / content.lines.count
-            ratio >= MINIFIED_AVG_LINE_LENGTH_CUTOFF
+            if content.lines.count != 0
+              ratio = content.chars.count / content.lines.count
+              ratio >= MINIFIED_AVG_LINE_LENGTH_CUTOFF
+            end
```

Comment:
I saw a comment on another PR this morning suggesting that a `?` method should probably return a bool, which made sense to me. In which case an `else... false` branch here might make sense.